AlfrescoApp/View Controllers/Settings View Controller/SettingsViewController.m

```diff
@@ -182,75 +184,155 @@ - (void)handleActionWithPreferenceIdentifier:(NSString *)preferenceIdentifier
+        }
+    }];
+}
+
+- (void) sendFeedbackHandler
+{
+    if ([MFMailComposeViewController canSendMail])
+    {
+        MFMailComposeViewController *emailController = [[MFMailComposeViewController alloc] init];
+        emailController.mailComposeDelegate = self;
+        [emailController setSubject:[self getFeedbackSubject]];
+        [emailController setToRecipients:@[kSettingsSendFeedbackAlfrescoRecipient]];
         
-        [resetAccountAlert showWithCompletionBlock:^(NSUInteger buttonIndex, BOOL isCancelButton) {
-            if (!isCancelButton)
-            {
-                // Reset accounts, delete cache databases, tmp folder, downloads
-                // Remove accounts
-                [[AccountManager sharedManager] removeAllAccounts];
-                // Delete cache
-                CoreDataCacheHelper *cacheHelper = [[CoreDataCacheHelper alloc] init];
-                [cacheHelper removeAllAvatarDataInManagedObjectContext:nil];
-                [cacheHelper removeAllDocLibImageDataInManagedObjectContext:nil];
-                [cacheHelper removeAllDocumentPreviewImageDataInManagedObjectContext:nil];
-                // Remove downloads
-                [[DownloadManager sharedManager] removeAllDownloads];
-                // Remove all contents of the temp folder
-                [[AlfrescoFileManager sharedManager] clearTemporaryDirectory];
-                
-                [[AnalyticsManager sharedManager] trackEventWithCategory:kAnalyticsEventCategorySettings
-                                                                  action:kAnalyticsEventActionClearData
-                                                                   label:kAnalyticsEventLabelFull
-                                                                   value:@1];
-                
-                UIAlertView *confirmation = [[UIAlertView alloc] initWithTitle:NSLocalizedString(@"settings.reset.confirmation.title", @"Reset Complete Title")
-                                                                       message:NSLocalizedString(@"settings.reset.confirmation.message", @"Reset Complete Message")
-                                                                      delegate:self
-                                                             cancelButtonTitle:NSLocalizedString(@"Close", @"Close")
-                                                             otherButtonTitles:nil];
-                [confirmation show];
-            }
-        }];
+        // Content body template
+        NSString *footer = [self getFeedbackFooter];
+        NSString *messageBody = [NSString stringWithFormat:@"<br><br>%@", footer];
+        [emailController setMessageBody:messageBody isHTML:YES];
+        emailController.modalPresentationStyle = UIModalPresentationPageSheet;
+        
+        [self presentViewController:emailController animated:YES completion:nil];
     }
 }
 
+#pragma mark - Feedback Utils
+
+- (NSString *) getFeedbackSubject
```

Comment:
[Apple's naming guidelines](https://developer.apple.com/library/mac/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingMethods.html#//apple_ref/doc/uid/20001282-1001751-BCIJHEDH) generally recommend not to use `get` in method names unless the values are returned indirectly (i.e. via passed-in pointers). Doesn't need changing here, but bear this in mind for future development. For example, a better method name might be `emailFeedbackSubject`